jaeseon/app.js

```diff
@@ -49,6 +48,48 @@ app.post('/users', async (req, res) => {
   return res.status(201).json({ message: 'userCreated' });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, user_id } = req.body;
```

Comment:
```suggestion
  const { title, content, userId } = req.body;
```
javascript에서는 snake_case가 아닌 camelCase로 작성하는것이 컨벤션입니다-!